libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,22 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix
+      [attr.aria-label]="'Search'">
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
+    <div class="book" data-testing="book-item" *ngFor="let b of (books$ | async)">
       <div class="book--title">
         {{ b.title }}
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img src="{{ b.coverUrl }}" alt= "{{ b.title }}"/>
```

Comment:
You have an extra space here between `alt=` and `"{{...`.

Also, less important but better for consistency, since you are using attribute binding syntax for aria labels, why not use it here as well?

Reply:
Also, I noticed that on file `libs/books/feature/src/lib/reading-list/reading-list.component.html` the `src` attribute uses the attribute binding syntax. I would change the `src` here to match the proper attribute binding syntax.

Reply:
Changed the attribute binding syntax.